### PR TITLE
TOMEE-2550-Translate to portuguese: examples/application-composer

### PR DESCRIPTION
**Pull request by Daniel-Dos**
This PR contains translate to portuguese and fix the title in the doc in english .

Link Jira : https://issues.apache.org/jira/browse/TOMEE-2550

### CODE REVIEW

File: examples/application-composer/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,164 @@
+= Compositor de Aplicações
+:index-group: Testing Techniques
+:jbake-type: page
+:jbake-status: published
+
+O `org.apache.openejb.junit.ApplicationComposer` é o corredor de testes JUnit modelado de acordo com a forma como fizemos testes internos no OpenEJB por anos (desde 2006).
```
Comment (marcoantoniobferreira) [inline examples/application-composer/README_pt.adoc]: Hi Daniel, in the sentence "corredor de testes" i believe that "executor de testes" make more sense

File: examples/application-composer/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+Isto não envolve o escaneamento de toda a classpath.  Se você quiser alguma coisa na aplicação, você deve construí-lo diretamente no seu testcase.
+
+Com o `ApplicationComposer` você pode fazer testes idênticos ao OpenEJB que usa internamente, mas com dependência limitada do próprio OpenEJB.  A principal dependência está no código
```
Comment (marcoantoniobferreira) [inline examples/application-composer/README_pt.adoc]: In this sentence: "você pode fazer testes idênticos ao OpenEJB que usa internamente" i believe that "você pode fazer testes idênticos ao que o OpenEJB  usa internamente"

**Comment by Daniel-Dos** [comment]
thanks for review Marco :  )

**Comment by Daniel-Dos** [comment]
thanks :  )

**Pull request merged**